AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -204,6 +204,8 @@ it-actual.ru##.blok[style="text-align: center;"]
 ! newsweek.com#@#.ad_wide
 ||cdn.pelcro.com/js/bab^
 digitaltrends.com#%#AG_abortInlineScript(/googlefc/, 'document.addEventListener');
+paraphrasing-tool.com#%#AG_abortInlineScript('whiteListRequestMessage', 'angular');
```

Comment:
I think it should be:
`paraphrasing-tool.com#%#AG_abortInlineScript(/whiteListRequestMessage/, 'angular');`